hack/build_custom_image-eviden.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script creates a custom container image for both, Barbican API and Barbican Worker,
+# including the HSM vendor's client software.
+# Vendor:  Eviden
+
+if [ "$#" -ne 5 ]; then
+    echo "Usage: $0 <registry_host> <namespace> <barbican-api_image_tag> <barbican-worker_image_tag> <eviden_iso_file>"
+    exit 1
+fi
```

Comment:
Shouldn't we have two registry hosts here - a source registry host (which should default to whatever the official RH registry is)  and a destination registry host - which would be your private registry?  These won't necessarily be the same.

Reply:
Understood.  I was looking for clarifying this but couldn't find any evidence.  Will fix it in the next push.